quickstarts/Tuning.ipynb

```diff
@@ -48,7 +48,7 @@
       "source": [
```

Comment:
<p>I wonder if we actually want to talk about the protos. I think we've been trying to focus on the higher-level abstractions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
**Protos** mentions removed